django/apps/existing_database/utils.py

```diff
@@ -0,0 +1,14 @@
+def parse_geom(geom):
+    # NOTE: This method transform data equivalent to what mapbox supports
+    # Since data structure from postgres isn't mapbox compatible
```

Comment:
not sure if we still need this. Do we use mapbox at some other part still?

Reply:
We have removed this.